flowgate-api/src/main/java/com/vmware/flowgate/controller/AssetController.java

```diff
@@ -103,6 +103,11 @@ public Asset getAssetByName(@PathVariable String name) {
       return assetRepository.findOneByAssetName(name);
    }
 
+   @RequestMapping(value = "/assetnumber/{number}", method = RequestMethod.GET)
+   public Asset getAssetByAssetNumber(@PathVariable Long number) {
```

Comment:
Let's use both the assetNumber and assetName to query the asset. 

Since assetNumber is not unique across different systems.

Reply:
Done